### PR TITLE
Add Windows info common build problems

### PR DESCRIPTION
**Pull request by ssabrii**

### CODE REVIEW

File: user/common-build-problems.md [hunk 1]
```diff
@@ -387,6 +387,11 @@ before_install:
 ```
 {: data-file=".travis.yml"}
 
+## **Windows**: common build problems and known issues
+
+For a list of common builds problems and known issues, please visit our community forum [here](https://travis-ci.community/t/current-known-issues-please-read-this-before-posting-a-new-topic/264).
```
Comment (MariadeAnton) [inline user/common-build-problems.md]:
```suggestion
For a list of common build problems on Windows, known issues and workarounds, please visit the [Travis CI community forum].(https://travis-ci.community/t/current-known-issues-please-read-this-before-posting-a-new-topic/264).
```

Comment (MariadeAnton) [reply user/common-build-problems.md]: wdyt @ssabrii ?

File: user/common-build-problems.md [hunk 2]
```diff
@@ -387,6 +387,11 @@ before_install:
...
+Our forum helps give better visibility on the issues customers are running into.
```
Comment (MariadeAnton) [inline user/common-build-problems.md]:
```suggestion
The [Travis CI community forum](https://travis-ci.community) provides better visibility on the issues customers are running into and how to solve them.
```

File: user/common-build-problems.md (hunk 1 above)
Comment (ssabrii) [reply user/common-build-problems.md]: Looks great thank you!

**Pull request merged**